README.md

```diff
@@ -44,278 +44,316 @@ Unrecommended ones can still be considered. The most important thing is to remai
--  🟢 idx • index
--  🟢 iface • interface
--  🟢 img • image
--  🟢 in • input
--  🟢 inc • include / increase
--  🟢 info • information
--  🟢 init • initialize
--  🟢 ins • insert
--  🟢 intf • interface
--  🟢 int • integer
--  🟢 iter • iterator
+-  identifier • 🟢 id
+-  image • 🟢 img
+-  include • 🟢 inc
+-  increase • 🟢 inc
+-  index • 🟢 idx
+-  information • info
+-  initialize • 🟢 init
+-  initialization • 🟢 init
+-  input 🟢 in • input
+-  insert • 🟢 ins
+-  integer • 🟢 int
+-  iterator • 🟢 iter
+-  interface
+   -  🟢 iface
+   -  🟢 intf
 
 ### J
 
--  🟢 js • javascript
+-  javascript • 🟢 js
 
 ### K
 
--  🟡 key • object key {only with **val**}
-
 ### L
 
--  🟢 lang • language
--  🟡 lat • latitude {only with **coord**}
--  🟢 lib • library
--  🟡 le • less or equal {binary operator}
--  🟢 len • length
--  🟢 ll • linked list
--  🟡 lon • longitude {only with **coord**}
--  🟡 lt • less than {binary operators}
+-  language • 🟢 lang
+-  latitude • 🟡 lat {only with **coord**}
+-  length • 🟢 len
+-  less or equal • 🟡 le {binary operator}
+-  less than • 🟡 lt {binary operator}
+-  library • 🟢 lib
+-  linked list • 🟢 ll
+-  longitude • 🟡 lon {only with **coord**}
 
 ### M
 
--  🟢 max • maximum
--  🟢 mcu • microcontroller
--  🟢 mem • memory
--  🟡 meta • metadata {data science}
--  🟢 mid • middle
--  🟢 min • minimum
--  🟢 misc • miscellaneous
--  🟢 mng • manager
--  🟢 mod • modulo
--  🟢 msg • message
--  🟡 mut • mutable {variables}
+-  manager • 🟢 mng
+-  maximum • 🟢 max
+-  memory • 🟢 mem
+-  message • 🟢 msg
+-  metadata • 🟡 meta {data science}
+-  microcontroller • 🟢 mcu
+-  middle • 🟢 mid
+-  minimum • 🟢 min
+-  miscellaneous • 🟢 misc
+-  modulo • 🟢 mod
+-  multiplier • 🟢 mplr
+-  mutable • 🟡 mut {variable}
 
 ### N
 
--  🔴 n • no {only with **yes**}
--  🟢 nav • navigation
--  🟡 ne • not equal {binary operator}
--  🟢 net • network
--  🟡 nl • newline {endcoding}
--  🟢 num • number
+-  navigation • 🟢 nav
+-  network • 🟢 net
+-  newline • 🟡 nl {endcoding}
+-  no • 🔴 n {only with **yes**}
+-  node 🟡 $... • {selector / DOM}
+-  not equal • 🟡 ne {binary operator}
+-  number • 🟢 num
 
 ### O
 
--  🟢 obj • object
--  🔴 ord • order {data science}
--  🟡 op • operation {binary operation}
--  🟢 opt • option / optional
--  🟢 os • operating system
--  🟢 out • output
+-  object • 🟢 obj
+-  object key • 🟡 key {only with **val**}
+-  operation • 🟡 op {binary operation}
+-  optional • 🟢 opt
+-  option • 🟢 opt
+-  operating system • 🟢 os
+-  order • 🔴 ord {data science}
+-  output • 🟢 out
 
 ### P
 
--  🟡 p • pointer {memory}
--  🟢 param • parameter
--  🟢 perf • performance
--  🟢 pic • picture
--  🟢 pos • position
--  🟢 pred • prediction
--  🟢 pref • preference
--  🟢 prev • previous
--  🟡 proc • process {thread}
--  🟢 prod • production
--  🟢 prof • profiler
--  🟢 ptr • pointer
--  🟢 pr • pull request
--  🟡 pwr • power {energy}
--  🟢 px • pixel
+-  parameter • 🟢 param
+-  performance • 🟢 perf
+-  picture • 🟢 pic
+-  pixel • 🟢 px
+-  pointer
+   -  🟡 p {memory}
+   -  🟢 ptr
```

Comment:
There are only a few places, where more than one option is added. Does it really wort breaking the format?

What about this:
pointer •  🟡 p {memory} • 🟢 ptr

Reply:
I think I will become hard to read.

Reply:
IMO all inline would nice and unified, and we should use it if possible. Can you imagine other formats other ideas for inlining?